invenio_drafts_resources/services/records/service.py

```diff
@@ -504,3 +504,12 @@ def _index_related_records(self, record, parent):
         #      instead (requires bulk indexing to work)
         for sibling in siblings:
             self.indexer.index(sibling)
+
+    def cleanup_drafts(self, timedelta_param):
```

Comment:
minor: should we call it just `timedelta`?
```suggestion
    def cleanup_drafts(self, timedelta):
```